e2e/tests/overview-page.spec.ts

```diff
@@ -217,13 +218,14 @@ test('Export to CSV button fails to download file and shows toast', async ({
   const exportButton = page.getByRole('button', {
     name: 'Export to CSV',
   });
-
   await expect(exportButton).toBeVisible();
   await exportButton.click();
 
   // Assert toast is visible
   const toast = page.locator('.toast');
   await toast.waitFor({state: 'visible'});
+
+  await page.unrouteAll();
```

Comment:
Were you seeing some weird behavior without this line? The routes created on the `page` should be isolated to the `test()` since the page fixture itself is set up and torn down for each test. [1]

[1] https://playwright.dev/docs/api/class-fixtures#fixtures-page

Reply:
I was not sure, and I was seeing strange behavior.  I was worried about this: "Once routing is enabled, every request matching the url pattern will stall unless it's continued, fulfilled or aborted." The docs on routes didn't mention that the unroute (or unrouteAll) call would not be needed after the end of a test.  But if one test throws an error before it completes, it would seem to affect others as well.  If we want to keep the unrouteAll() call, it should probably be in the afterTest method, which I believe does get called even if the test throws.

I did try calling `await page.unrouteAll()` immediately before the route() call, and that seemed to make things much worse, which seems wrong.